static/js/context-menu.js

```diff
@@ -54,14 +54,18 @@ function symbolsFromString(symbols) {
   return symbols.split(",");
 }
 
-function findReferences(symbols) {
+function findReferences(symbols, visibleToken) {
   symbols = symbolsFromString(symbols);
   if (!symbols.length)
     return $();
   symbols = new Set(symbols);
   return $([...document.querySelectorAll("span[data-symbols]")].filter(span => {
+    if (span.textContent != visibleToken) {
```

Comment:
Maybe just `return span.textContent == visibleToken && ...`, though no big deal in any case.

Reply:
Updated.